### PR TITLE
[AKS] az aks: Add aad session key support. DO NOT MERGE

### PR DESCRIPTION
**Pull request by JunSun17**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by JunSun17** [comment]
Pending integration test.

**Comment by yonzhan** [comment]
AKS

**Comment by qwordy** [comment]
I am not very familiar with background. Can you share what's the effect to users?

**Comment by yungezz** [comment]
hi @JunSun17 pls ping us at teams channel when the PR ready to merge.

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -219,13 +219,15 @@ def _get_default_dns_prefix(name, resource_group_name, subscription_id):
 
 
 # pylint: disable=too-many-locals
-def store_acs_service_principal(subscription_id, client_secret, service_principal,
+def store_acs_service_principal(subscription_id, client_secret, service_principal, aad_session_key,
```
Comment (arrownj) [inline src/aks-preview/azext_aks_preview/custom.py:222]: Is this a new parameter of some command ? If yes, could you please also add argument definition such as help message for it ?

**Pull request closed without merging**